src/zeta/aca_oam_port_manager.cpp

```diff
@@ -99,60 +99,56 @@ void Aca_Oam_Port_Manager::_clear_all_data()
 void Aca_Oam_Port_Manager::create_entry_unsafe(uint port_number)
 {
   ACA_LOG_DEBUG("%s", "Aca_Oam_Port_Manager::create_entry_unsafe ---> Entering\n");
-
-  unordered_set<uint> tunnel_ids_table;
-  _oam_ports_table.emplace(port_number, tunnel_ids_table);
+  
+  _oam_ports_cache.emplace(port_number);
 
   ACA_LOG_DEBUG("%s", "Aca_Oam_Port_Manager::create_entry_unsafe <--- Exiting\n");
 }
 
 // update oam_ports_table and add the OAM punt rule also if this is the first port in the VPC
-void Aca_Oam_Port_Manager::add_vpc(uint port_number, uint tunnel_id)
+void Aca_Oam_Port_Manager::add_port_number(uint port_number)
```

Comment:
update function name as this function is doing more than just add_port_number in _oam_ports_cache

suggested function name: add_port_rule